### PR TITLE
[SRM-954] Added changes related to Drupal 10 upgrade.

### PR DESCRIPTION
**Pull request by MdNadimHossain**
### JIRA
https://digital-vic.atlassian.net/browse/SRM-954

### Changes
D10 compatibility changes.

### CODE REVIEW

File: src/TideNewsOperation.php [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+ */
+class TideNewsOperation {
+
+  /**
+   * Enable editorial workflow and shceduled transitions.
+   */
+  public static function enableNecessaryModules() {
+    // Enable Editorial workflow if workflow module is enabled.
+    $moduleHandler = \Drupal::service('module_handler');
+    if ($moduleHandler->moduleExists('workflows')) {
```
Comment (krakerag) [inline src/TideNewsOperation.php:19]: Let's get in the habit of early return on these module present checks. No need to nest large operations within a module check, just exit early.

This also means I would put the work for each in seperate functions (lines 17 to 25 in one function, lines 28 to 46 in another function) so you can perform early return on both module checks properly

File: tide_news.install [hunk 2]
```diff
@@ -11,6 +11,6 @@ use Drupal\tide_news\TideNewsOperation;
  * Implements hook_install().
  */
 function tide_news_install() {
-  TideNewsOperation::enableNecessaryModules();
-  TideNewsOperation::assignNecessaryPermissions();
+  TideNewsOperation::addToWorkflows();
+  TideNewsOperation::addToShceduledTransitions();
```
Comment (krakerag) [inline tide_news.install:15]: Typo

Comment (MdNadimHossain) [reply tide_news.install:15]: yup, by mistake removed it. Fixed now.

File: tide_news.install [hunk 3]
```diff
@@ -5,801 +5,13 @@
...
-          'autocomplete' => FALSE,
-          'width' => '100%',
-        ];
-      }
-    }
-    $config->set('content', $content);
-    $config->save();
-  }
+  TideNewsOperation::addToWorkflows();
+  TideNewsOperation::addToShceduledTransitions();
```
Comment (krakerag) [inline tide_news.install:15]: Still a typo here :(

`addToShceduledTransitions` should be `addToScheduledTransitions`

Comment (MdNadimHossain) [reply tide_news.install:15]: my god, I am going blind. Good one. Let me fix it for one more time :D

**Review by krakerag — APPROVED** [review]
🚀

**Pull request merged**